### DISCUSSION TITLE
Welcome to omegaide Discussions!

### DISCUSSION THREAD

**Discussion by omegaui** [omegaui/omegaide#2] [General]
## Ask Questions or give a suggestion or report a bug or just hangout

**Comment by kvnandula04** [comment]
Hey, just saw your comment on Bobbylight's RSyntaxTextArea which brought me here. Did you make this entirely out of RSyntaxTextArea? Seems awesome!

**Reply by omegaui** [reply]
Thank you!!

Yes I made that Editor with some changes in the theming system of RSyntaxTextArea Sources

**Reply by omegaui** [reply]
You change the theming either by editing the sources or the theme.xml files located at **org.fife.ui.rsyntaxtextarea.themes** directory

**Reply by kvnandula04** [reply]
Ohk, that's actually a problem I'm having. I added the RSTA as a jar to my project and I'm unable to edit anything in it. Do you know how I can edit it? I'd like to change the default themes but it says "File is read only". 

So I added the jar under Project Structure - Modules.
![image](https://user-images.githubusercontent.com/34941351/105640729-5c719980-5e77-11eb-8085-74e9ba20d85f.png)

**Reply by omegaui** [reply]
**You cannot edit ByteCode in Java that's read only**

The only way to edit it is to remove that jar dependency from your project.

Download the rsta sources from github release section or just clone it

Copy the sources located at main/java to your source folder
And the folder located at the main/resources to your resource root.

And then you can easily edit it.

Well, From which country you are?
It looks like we are living in different time zones.

😇

**Reply by kvnandula04** [reply]
Hey,

Sorry, just saw this. Could you take screenshots by what you mean, please? I don't quite understand it. Sorry.

I'm from the UK btw, what about you?

**Reply by omegaui** [reply]
I am from India.

Sorry, but it is quite time consuming to take the screenshots.

What I can tell you is.

**Clone the Repository First**

**Reply by kvnandula04** [reply]
Hey! Thanks for the quick response. Do you have a social media app to connect with? I'm not active on here, so it would speed things up if we do it on a social media platform or something... Then you can guide me and I can try it out instantaneously. What do you say? My Twitter is [@NandulaKarthik](https://twitter.com/NandulaKarthik).

**Reply by omegaui** [reply]
Great!
⚡⚡⚡
But,
I use instagram only.
My Instagram is @i_am_arham_92

**Reply by kvnandula04** [reply]
Awesome! Just sent a dm.
